cinema/management/wait_for_db.py

```diff
@@ -0,0 +1,20 @@
+import time
+
+from django.core.management import BaseCommand
+from django.db import connections, OperationalError
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        self.stdout.write("Waiting for database...")
+        connection = None
+        while not connection:
+            try:
+                connection = connections["default"]
+                connection.cursor().execute("SELECT 1")
+            except OperationalError:
+                self.stdout.write("Database unavailable, waiting 1 second...")
+                connection = None
+                time.sleep(1)
```

Comment:
This loop will continue indefinitely if the database is not available. It would be better to add a maximum number of attempts to prevent the script from hanging indefinitely. You can use a for loop with a range instead of a while loop to achieve this.